src/NumericalAlgorithms/LinearOperators/PowerMonitors.hpp

```diff
@@ -45,4 +45,64 @@ std::array<DataVector, Dim> power_monitors(const DataVector& input_data_vector,
                                            const Mesh<Dim>& mesh);
 /// @}
 
+/// @{
+/*!
+ * \ingroup SpectralGroup
+ * \brief Compute the negative log10 of the relative truncation error.
+ *
+ * Truncation error according to Eqs. (57) and (58) of Ref.
+ * \cite Szilagyi2014fna, i.e.,
+ *
+ * \f{align*}{
+ *  \mathcal{T}\left[P_k\right] = \log_{10} \max \left(P_0, P_1\right)
+ *   - \dfrac{\sum_{j} \log_{10} \left(P_j\right) w_j }{\sum_{j} w_j} ,
+ * \f}
+ *
+ * with weights
+ *
+ * \f{align*}{
+ *  w_j = \exp\left[ - (j - N_k + \dfrac{1}{2})^2 \right] .
+ * \f}
+ *
+ * where \f$ N_k \f$ is the number of modes or gridpoints in dimension k. Here
+ * the second term is a weighted average with larger weights toward the highest
+ * modes. The negative of this number should correspond to the number of digits
```

Comment:
Isn't this number always positive (or should be unless the modes are doing something weird)?

Reply:
Yes, it is positive. I'll correct the text when I push the changes for the other comments.